lua/guess-indent/init.lua

```diff
@@ -266,6 +266,13 @@ end
 -- The argument `context` should only be set to `auto_cmd` if this function gets
 -- called by an auto command.
 function M.set_from_buffer(context)
+  if not config.override_editorconfig then
```

Comment:
This if statement should probably be moved down into the `if context == "auto_cmd" then` block. That way, guess-indent won't override the editorconfig when it gets launched by the auto command, but it will override it if called manually using `:GuessIndent`.
I feel like this closer to the behaviour a user would expect.
Do you have any objections?

Reply:
Yeah this seems like a sane default. I just pushed a commit to modify this and updated the `doc` file to explain this as well. :D